core/src/Kokkos_View.hpp

```diff
@@ -170,7 +170,6 @@ class View : public Impl::BasicViewFromTraits<DataType, Properties...>::type {
   using data_type            = DataType;
   using const_data_type      = typename traits::const_data_type;
   using non_const_data_type  = typename traits::non_const_data_type;
-  using view_tracker_type    = Impl::ViewTracker<View>;
```

Comment:
Why is this removed? Seems like an unrelated change.

Reply:
Good catch!